packages/Typography/src/Text.tsx

```diff
@@ -27,10 +27,10 @@ export const Text: FC<TextProps> = forwardRef((props: TextProps, ref: RefObject<
         [styles.h4]: size === 4,
         [styles.h5]: size === 5,
         [styles.h6]: size === 6,
-        [styles.subheading]: size === 7,
-        [styles.paragraph]: size === 8,
-        [styles.meta]: size === 9,
-        [styles.tooltip]: size === 10,
+        [styles.subheading]: size === 'subhead',
+        [styles.paragraph]: size === 'body',
```

Comment:
Don't forget to change your default `size` (line 20) since `8` is no longer used

Reply:
Nice catch :+1: Thank you!